Sources/HaCWebsiteLib/ViewModels/ImageHero.swift

```diff
@@ -18,12 +38,36 @@ struct ImageHero {
     case color(String)
   }
 
+  enum Color {
+    case dark
+    case light
+  }
+
+  private var dateBlock: Node? {
+    return formattedDate.map({ dateString in El.Div[Attr.className => "ImageHero__date"].containing(dateString) })
```

Comment:
This is still a little unergonomic

Reply:
We talking about HaCTML here or the `map` construct I'm using?

Reply:
Bit of both
Think optionals make sense in general for Nodes that may or may not exist but the mapping is not very intuitive so we should explore other options for this. Nathan brought this up too. Not something that needs to be addressed right now